recipe/build.sh

```diff
@@ -1,27 +1,33 @@
 #!/bin/bash
 
+set -eo pipefail
+
 declare -a CMAKE_PLATFORM_FLAGS
-if [[ ${HOST} =~ .*linux.* ]]; then
-    CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
-fi
 
 mkdir build_shared && cd $_
-cmake \
+cmake ${CMAKE_ARGS} \
+    -DALLOW_IN_SOURCE_BUILD=ON \
     -DCMAKE_PREFIX_PATH=$PREFIX \
     -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON \
+    -DJAS_STDC_VERSION="201112L" \
     -DJAS_ENABLE_AUTOMATIC_DEPENDENCIES=False \
-    ${CMAKE_PLATFORM_FLAGS[@]} ..
-make
+    -DJAS_ENABLE_LIBHEIF=OFF \
+    -DJAS_ENABLE_LATEX=OFF \
+    "${CMAKE_PLATFORM_FLAGS[@]}" ..
+make -j${CPU_COUNT}
 make install
 cd ..
 
 mkdir build_static && cd $_
```

Comment:
Can you remove the static build completely?
From the build logs, static binaries are overriding shared binaries.

Reply:
After removing the static build I got an error on `osx` platforms:
`   ['  ERROR (jasper,bin/jasper): /private/var/folders/c_/qfmhj66j0tn016nkx_th4hxm0000gp/T/abs_d8u9w5rmfj/croot/jasper_1713880906898/work/build_shared/src/libjasper/libjasper.7.dylib not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (jasper,bin/jiv): /private/var/folders/c_/qfmhj66j0tn016nkx_th4hxm0000gp/T/abs_d8u9w5rmfj/croot/jasper_1713880906898/work/build_shared/src/libjasper/libjasper.7.dylib not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (jasper,bin/imgcmp): /private/var/folders/c_/qfmhj66j0tn016nkx_th4hxm0000gp/T/abs_d8u9w5rmfj/croot/jasper_1713880906898/work/build_shared/src/libjasper/libjasper.7.dylib not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (jasper,bin/imginfo): /private/var/folders/c_/qfmhj66j0tn016nkx_th4hxm0000gp/T/abs_d8u9w5rmfj/croot/jasper_1713880906898/work/build_shared/src/libjasper/libjasper.7.dylib not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?'] `

Reply:
I don't get this locally. Maybe try to restart the builds and see if it fixes the problem?